_topic_map.yml

```diff
@@ -739,28 +739,115 @@ Topics:
+  - Name: Installing the QEMU guest agent on virtual machines
+    File: cnv-installing-qemu-guest-agent
+  - Name: Viewing the IP address of vNICs on a virtual machine
+    File: cnv-viewing-ip-of-vm-vnic
+### Advanced virtual machine configuration
+  - Name: Configuring PXE booting for virtual machines
+    File: cnv-configuring-pxe-booting
+  - Name: Managing guest memory
+    File: cnv-managing-guest-memory
+### Templates
+  - Name: Creating virtual machine templates
+    File: cnv-creating-vm-template
+  - Name: Editing a virtual machine template
+    File: cnv-editing-vm-template 
+  - Name: Deleting a virtual machine template
+    File: cnv-deleting-vm-template 
+### Importing virtual machines
+  - Name: Importing virtual machine images with DataVolumes
+    File: cnv-importing-virtual-machine-images-datavolumes
+  - Name: Importing a VMware virtual machine to your Container-native Virtualization cluster
+    File: cnv-importing-vmware-vm-to-ocp
+### Cloning virtual machines
+  - Name: Cloning a virtual machine disk into a new DataVolume
+    File: cnv-cloning-vm-disk-into-new-datavolume
+  - Name: Cloning a virtual machine by using a DataVolumeTemplate
+    File: cnv-cloning-vm-using-datavolumetemplate
+### A BETTER NAME THAN 'STORAGE 4 U'
+  - Name: Uploading local disk images by using the virtctl tool
+    File: cnv-uploading-local-disk-images-virtctl
+  - Name: Expanding virtual storage by adding blank disk images
+    File: cnv-expanding-virtual-storage-with-blank-disk-images
+### Virtual machine live migration
+  - Name: Virtual machine live migration
+    File: cnv-live-migration
+  - Name: Live migration limits and timeouts
+    File: cnv-live-migration-limits
+  - Name: Migrating a virtual machine instance to another node
+    File: cnv-migrate-vmi
+  - Name: Monitoring live migration of a virtual machine instance
+    File: cnv-monitor-vmi-migration
+  - Name: Cancelling the live migration of a virtual machine instance
+    File: cnv-cancel-vmi-migration
+### Node maintenance mode
+  - Name: Node maintenance mode
+    File: cnv-node-maintenance
+  - Name: Configuring virtual machine eviction strategy
+    File: cnv-configuring-vmi-eviction-strategy
+  - Name: Setting a node to maintenance mode
+    File: cnv-setting-node-maintenance
+  - Name: Resuming a node from maintenance mode
+    File: cnv-resuming-node
+### Installing VirtIO drivers on Windows virtual machines
+  - Name: Installing VirtIO driver on an existing Windows virtual machine
+    File: cnv-installing-virtio-drivers-on-existing-windows-vm
+  - Name: Installing VirtIO driver on a new Windows virtual machine
+    File: cnv-installing-virtio-drivers-on-new-windows-vm
+### Logging, events, and monitoring
+  - Name: Viewing logs
+    File: cnv-logs
+  - Name: Viewing events
+    File: cnv-events
+  - Name: OpenShift cluster monitoring
+    File: cnv-openshift-cluster-monitoring
 #- Name: Container-native Virtualization Release Notes
```

Comment:
@bergerhoffer noted that the guide titles should be in sentence case, so when this gets merged with the release notes, please make sure it days "Container-native Virtualization release notes." I'm changing the {RN_bookname} in my PR to reflect this. cc @vikram-redhat